SignalRServiceBenchmarkPlugin/common/Extensions.cs

```diff
@@ -32,6 +34,22 @@ public static class Extensions
             return false;
         }
 
+        public static void TryGetTypedValue<TKey, TValue, TActual>(this IDictionary<TKey, TValue> dict, TKey key, out TActual val, Func<TValue, TActual> converter = null) where TActual : TValue
+        {
+            var success = dict.TryGetTypedValueInternal(key, out val, converter);
+            HandleGetValueResult(success, key.ToString());
+        }
+
+        private static void HandleGetValueResult(bool success, string key)
```

Comment:
This function is so simple and tight related to TryGetTypedValue(). Consider to merge them.

Reply:
Got it, thanks.

Reply:
Merged.